lib/app_web/templates/page/index.html.eex

```diff
@@ -1,54 +1,59 @@
-<section class="tc pb4">
-  <h1>Github Backup</h1>
-  <h2 class="f2">Backup all of your issues for offline use.</h2>
-</section>
-
-<section class="pb4 tl relative w-90 w-50-ns center">
-  <div class="dib br2 mr3 mv0 absolute top-0 left-0">
-    <img class="br2 w3"src="./images/user_avatar.png" alt="username image">
-  </div>
-  <div class="tl dib ba gh-b--light-gray br2 relative comment ml5-5 comment-minw">
-    <p class="pa3 bb gh-b--light-gray bg-light-gray gh-gray mb0 cf">
-      <strong>username </strong>
-      commented time-frame ago
-      <span class="ba br2 pa1 f4 b fr gh-b--light-gray lh-1-25">Member</span>
-    </p>
-    <p class="pa3 mv0 lh-copy">
-      This is the most recent comment.
+<section class="bg-light-gray">
+  <div class="pv3 pv6-ns w-80 w-75-ns center">
+    <h2 class="f-subheadline-ns f1 fw8">
+      Keep the history<br>
+      of your issues<br>
+      and comments.
+    </h2>
+    <p class="f2 pv4 pr5">
+      <strong class="gb-light-teal fw8">Gitbu</strong>
+      has been created so you never lose track of the history of your issues
+      and comments. Find out what was added, when and by who.
     </p>
   </div>
 </section>
 
-<section class="pb4 tl relative w-90 w-50-ns center">
-  <div class="dib br2 mr3 mv0 absolute top-0 left-0">
-    <img class="br2 w3"src="./images/user_avatar.png" alt="username image">
+<section class="pv4 pv6-ns w-80 center tc f2">
+<div class="w-30-ns dib-ns tc v-top ph4 pv4 pv0-ns">
+  <div class="w-100 center tc">
+    <i class="fas fa-arrow-circle-down"></i>
   </div>
-  <div class="tl dib ba gh-b--light-gray br2 relative comment ml5-5 comment-minw">
-    <p class="pa3 bb gh-b--light-gray bg-light-gray gh-gray mb0 cf">
-      <strong>username </strong>
-      commented time-frame ago
-      <span class="ba br2 pa1 f4 b fr gh-b--light-gray lh-1-25">Member</span>
-    </p>
-    <p class="pa3 mv0 lh-copy bg-moon-gray">
-      This was the comment before the latest version of the comment, it was a
-      bit longer than the latest comment so you can see the size of the box. If
-      I add to it even more it still looks great!
-    </p>
+  <h4 class="lh-copy">DOWNLOAD THE APP</h4>
```

Comment:
"Install Gitbu" instead?

Reply:
Looks like install is the right terminology:
![image](https://user-images.githubusercontent.com/16775804/37608699-22bd6648-2b93-11e8-87d7-5a18bcba828f.png)
So I'd be happy with that copy change...